galaxy_ng/tests/integration/api/test_synclist.py

```diff
@@ -132,35 +132,34 @@ def test_synclist_object_and_synclist_repo_edit(ansible_config, upload_artifact)
     synclist_name = synclist_data_before["name"]
     synclist_id = synclist_data_before["id"]
 
-    # check that collection is part of synclist repo
+    # check collection in viewset {synclist_name}/v3/collections/
     url = f"content/{synclist_name}/v3/collections/?limit=30"
     resp = api_client(url)
     collections_before = [(c["namespace"], c["name"]) for c in resp["data"]]
     assert collection_key in collections_before
 
-    # edit synclist payload
+    # check collection not in viewset {synclist_name}/v3/excludes/
+    url = f"content/{synclist_name}/v3/excludes/"
+    resp = api_client(url)
+    excludes = [(c["name"].split(".")[0], c["name"].split(".")[1]) for c in resp["collections"]]
```

Comment:
So the new return from /excludes/ is a list of namestrings  ie  ['somenamespace.somename', 'cisco.someciscothing'] ?

Just verifying that is intentional and ok (same as comments above).

Reply:
Might be easier to make collection_key just a string "somenamespace.somename"
and ```asssert collection_key in resp["collections"]```  or some variant. But doesn't really matter.

Reply:
excludes currently returns this (and it this PR won't change that, it just changes how it is looked up)
```
    "collections": [
        {
            "name": "autohubtest2.collection_dep_a_boegefmx"
        },
    ]
```

Reply:
was thinking tests might be clearer with

``` python
collection_key = "autohubtest2.collection_dep_a_boegefmx"
assert "collections" in resp
assert "name" in resp["collections"]
assert collection_key == resp["collections"]["name"]
```

Reply:
it ends up being a list of dictionaries each with a name key, so i think the list comprehension is needed
```
    "collections": [
        {
            "name": "redhat.satellite"
        },
        {
            "name": "ibm.ibm_zos_cics"
        },
```